configuration/configuration.go

```diff
@@ -646,6 +635,40 @@ type Proxy struct {
 	Password string `yaml:"password"`
 }
 
+type Validation struct {
+	// Enabled enables the other options in this section. This field is
+	// deprecated in favor of Disabled.
+	Enabled bool `yaml:"enabled,omitempty"`
+	// Disabled disables the other options in this section.
+	Disabled bool `yaml:"disabled,omitempty"`
+	// Manifests configures manifest validation.
+	Manifests ValidationManifests `yaml:"manifests,omitempty"`
+}
+
+type ValidationManifests struct {
+	// URLs configures validation for URLs in pushed manifests.
+	URLs struct {
+		// Allow specifies regular expressions (https://godoc.org/regexp/syntax)
+		// that URLs in pushed manifests must match.
+		Allow []string `yaml:"allow,omitempty"`
+		// Deny specifies regular expressions (https://godoc.org/regexp/syntax)
+		// that URLs in pushed manifests must not match.
+		Deny []string `yaml:"deny,omitempty"`
+	} `yaml:"urls,omitempty"`
+	// ImageIndex configures validation of image indexes
+	ImageIndex ValidationManifestsImageIndex `yaml:"imageindex,omitempty"`
+}
+
+type ValidationManifestsImageIndex struct {
+	Images struct {
+		// Exist validates that images referenced by indexes exist in the registry
+		// before the image index is accepted.
```

Comment:
after reading the docs below, i think this godoc needs to be improved to explain how Exists and Platforms interact.  But i'm also not loving the overall config api (i'll comment on that elsewhere)